src/services/postService.js

```diff
@@ -15,19 +16,46 @@ async function seePosts(){
     return posts.Items;
 }
 
-async function createReply(username, text, id){
+async function createReply(userID, text, id){
     const post = await postDAO.getPost(id);
     if (!post.Item) {
-        throw {status: 400, message: "That post doesn't exist"};
+        throw {status: 400, message: `Post ${id} doesn't exist`};
     }
-    const reply = [{postedBy: username, description: text}];
+    const reply = {postedBy: userID, description: text, itemID: uuid.v4()};
```

Comment:
Add main thread id. postID: id or mainPost: id

Reply:
I thought about that but, I decided replies should be stored in the post themselves in a list. If you needed to delete them, just go to the post and remove it from the list. At least, that's how I implemented it.

Reply:
I didn't realize you aren't actually creating a table entry in the database. It just goes straight into the post replies